File: leetcode-python/String/面试题 01.01-判定字符是否唯一.py
'''
输入: s = "leetcode"
输出: false
'''


class Solution:
    def isUnique(self, s: str) -> bool:
        d = {}
        for i in s:
            if i in d:
                # d[i] += 1
                return False
            else:
                d[i] = 1
        return True

File: leetcode-javascript/string/面试题 01.01-判定字符是否唯一.js
var isUnique = function(astr) {
    let map = {}
    for (let c of astr) {
        map[c] = (map[c] || 0) + 1
    }

    for (let key in map) {
        if (map[key] > 1) {
            return false
        }
    }
    return true
};

